youtube_dl/extractor/itunes.py

```diff
@@ -0,0 +1,76 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import datetime
+import re
+
+from .common import InfoExtractor
+
+from ..utils import (
+    unescapeHTML,
+    int_or_none,
+    unified_strdate,
+)
+
+
+class iTunesIE(InfoExtractor):
+    _VALID_URL = r'https?://itunes\.apple\.com/(?P<lang>[a-z]{2})/(?P<genre>[a-z0-9-]+)/?(?P<display_id>[a-z0-9-]+)?/(?:id)?(?P<id>[0-9]+)'
+    _TEST = {
+        'url': 'https://itunes.apple.com/us/itunes-u/uc-davis-symphony-orchestra/id403834767',
+        'info_dict': {
+            'id': '403834767',
+            'title': 'UC Davis Symphony Orchestra & University Chorus',
+        },
+        'playlist_count': 31,
+    }
+
+    _USER_AGENT = "iTunes/10.2.1 (Macintosh; Intel Mac OS X 10.7) AppleWebKit/534.20.8"
+
+    def html_tags_to_dict(self, string):
```

Comment:
Seems this function is doing a similar work to [```extract_attributes()```](https://github.com/rg3/youtube-dl/blob/293c255/youtube_dl/utils.py#L302-L321).